apps/frontend/src/components/views/election/ballotPaperBoard/ballotPaperSection/BallotPaperSectionSettingsMenu.tsx

```diff
@@ -42,6 +55,23 @@ export const BallotPaperSectionSettingsMenu = ({
         electionId={electionId}
         ballotPaperSection={ballotPaperSection}
       />
+      <MutateBallotPaperSectionDrawer
+        title={'Edit ballot paper section'}
+        opened={mutateSectionContextOpened}
+        onClose={mutateSectionContextActions.close}
+        mutateButtonText={'Save changes'}
+        onMutate={onMutate}
+        isMutating={isMutating}
+      />
```

Comment:
You are missing the ballot paper section as parameter here.
Because of this the current values aren't displayed when editing a ballot paper section.
```suggestion
      <MutateBallotPaperSectionDrawer
        ballotPaperSection={ballotPaperSection}
        title={'Edit ballot paper section'}
        opened={mutateSectionContextOpened}
        onClose={mutateSectionContextActions.close}
        mutateButtonText={'Save changes'}
        onMutate={onMutate}
        isMutating={isMutating}
      />
```